packages/TelegramClient-Core.package/TCCPhotoMessage.class/class/newFrom..st

```diff
@@ -0,0 +1,13 @@
+as yet unclassified
+newFrom: aMessageContent
+
+	| photoData |
+	photoData := ((aMessageContent at: 'photo') at: 'sizes') last.
+
+	^ self
+		new
```

Comment:
I would leave the `new` on the previous line as the result of that is the subject of the cascade

Reply:
fixed